Spring-Boot-News-Service/src/main/java/com/atomshermental/springbootnewsservice/NewsRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.atomshermental.springbootnewsservice;
+
+import com.atomshermental.springbootnewsservice.model.Comment;
+import com.atomshermental.springbootnewsservice.model.News;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@AllArgsConstructor
+@Repository
+public class NewsRepository {
```

Comment:
You should make an interface of this repository (otherwise you won't be able to use mocks)